satellite-clone-vars.sample.yml

```diff
@@ -1,41 +1,40 @@
 ---
+# Throughout this documentation, ensure that you understand the following terminology:
+# - Source server: Existing Satellite server.
+# - Target server: new server, to which Satellite server is being cloned to.
+# Note:
+# 1. All the options mentioned in this file affects only the target server.
+# 2. To change any variable, just uncomment the corresponding line and provide a new value.
+
 ############## optional variables ####################
-#
-# The backup folder on your Destination node.
-# NOTE: Place the backup files only on the Destination node and not on the Ansible Control node.
-#
-#   Required backup files:
+
+# The backup folder on your target server (defaults to /backup).
+# Required backup files:
 #   - Standard backup scenario : config_files.tar.gz, mongo_data.tar.gz, pgsql_data.tar.gz, (optional) pulp_data.tar
 #   - Online backup or RHEL migration scenario   : config_files.tar.gz, mongo_dump folder, foreman.dump, candlepin.dump
-#
-#    If the folder mentioned by `backup_dir` does not exist or not accessible, the script will stop with an error.
-#
-# uncomment the following line to override this option (defaults to /backup)
+# Note:
+# 1. Place the backup files only on the target server and not on the source server.
+# 2. If the folder mentioned by `backup_dir` does not exist or not accessible, the script will stop with an error.
 #backup_dir: /backup
 
-# Misc info
-# Disable firewall - This setting can optionally be used to disable firewall to setup Satellite for testing purposes.
-# uncomment the following line to override this option (defaults to false)
+# This can be optionally (defaults to false) used to disable firewall to setup Satellite for testing purposes.
 #disable_firewall: false
 
-# Run katello-reindex - This setting can optionally be used to run katello reindex after the Satellite install.
+# This can be optionally (defaults to false) used to run katello reindex after the Satellite install.
```

Comment:
I know you didn't add this, but we can probably remove the `This can be optionally` parts in this file as all variables are optional and we say "optional variables" at the top.

Reply:
@johnpmitsch added a new temporary commit, let me know what you think